openslides/agenda/static/js/agenda/base.js

```diff
@@ -23,21 +23,26 @@ angular.module('OpenSlidesApp.agenda', ['OpenSlidesApp.users'])
 
 .factory('AgendaUpdate',[
     'Agenda',
-    function(Agenda) {
+    'operator',
+    function(Agenda, operator) {
         return {
             saveChanges: function (item_id, changes) {
-                Agenda.find(item_id).then(function(item) {
-                    var something = false;
-                    _.each(changes, function(change) {
-                        if (change.value !== item[change.key]) {
-                            item[change.key] = change.value;
-                            something = true;
+                // change agenda item only if user has the permission to do that
+                if (operator.hasPerms('agenda.can_manage agenda.can_see_hidden_items')) {
+                    Agenda.find(item_id).then(function (item) {
```

Comment:
FYI: We cannot get rid of this find: This function is always called if a motion, topic, assignment is created to set properties of the also created agenda item. But the autoupdate is not fast enough to deliver the new agenda item, so a `Agenda.get()` always return undefined. So we have to wait for this item.